index.html

```diff
@@ -11,6 +12,72 @@
 </head>
 
 <body>
-<!-- Use past code you created here for Great Idea Web Page or past solution for Great Idea Web Page HTML here! -->
+<!-- Copy and paste your HTML from the first UI project here -->
+<div class="container">
+  <header>
+    <nav>
+      
+      <a href="./services.html">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+      <a id="logo1" href="#"><img class="logo" src="img/logo.png" alt="Great Idea! Company logo."></a>
+      
+    </nav>
+    
+  
+  </header>
+    
+  <section class="cta">
+    <div class="title">
+      <h1>Innovation<br>On<br>Demand</h1>
+      <button class="headerButton">Get Started</button>
+    </div>
+    <img src="img/header-img.png" alt="Image of a code snippet.">
+  </section>
+  <hr>
+  <section class="main-content">
+    <div class="above-image">
+      <div>
+        <h4>Features</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+      <div>
+        <h4>About</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+    </div>
+    <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+    <div class="below-image">
+      <div>
+        <h4>Services</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>    
+      </div>
+      <div>
+        <h4>Product</h4>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>
+      </div>
+      <div>
+        <h5>Vision</h5>
+        <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.</p>    
+      </div>
+    </div>
+  </section>
+  <hr>
```

Comment:
same here
these should be border top/bottom of the div/section you have them separating